app/Http/Controllers/Settings/GendersController.php

```diff
@@ -104,4 +129,27 @@ public function destroy(GendersRequest $request, Gender $gender)
 
         return $this->respondObjectDeleted($gender->id);
     }
+
+    /**
+     * Update the given gender to the default gender.
+     */
+    public function updateDefault(GendersRequest $request, Gender $gender)
+    {
+        $account = auth()->user()->account;
+        $account->default_gender_id = $gender->id;
+        $account->save();
+
+        return $this->format($gender);
+    }
+
+    private function format($gender)
```

Comment:
`format` is not a good name for this method. Also, no documentation for this function?

Reply:
I've renamed it `formatData`